examples/sn-dms-demo/package.json

```diff
@@ -84,6 +84,7 @@
     "@types/lodash": "^4.14.119",
     "@types/lodash.groupby": "^4.6.4",
     "@types/node": "^10.12.12",
+    "@types/path-to-regexp": "^1.7.0",
```

Comment:
path-to-regexp is not even a dependency of this package. I think that should be added as a dependency without the types package.